imports/ui/components/search-bar.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import TextField from 'material-ui/TextField'
+import {
+  textInputFloatingLabelStyle,
+  textInputUnderlineFocusStyle,
+  whiteInput
+} from '../components/form-controls.mui-styles'
+
+export default class SearchBar extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      searchText: ''
+    }
+  }
+
+  handleSearch = (evt) => {
+    this.setState({searchText: evt.target.value})
+    this.props.findItem(evt.target.value)
+  }
+
+  render () {
+    const { searchText } = this.state
+    return (
+      <TextField
+        hintText='Search units'
+        floatingLabelShrinkStyle={textInputFloatingLabelStyle}
+        underlineFocusStyle={textInputUnderlineFocusStyle}
+        inputStyle={whiteInput}
+        hintStyle={whiteInput}
+        fullWidth
+        value={searchText}
+        onChange={(evt) => this.handleSearch(evt)}
+      />
+    )
+  }
+}
```

Comment:
I resorted to the search bar component since putting directly Textfield seems to override hintText behavior. HintText would not disappear once a user starts to type). I also tried placeholder but changing the font color seems a bit complex. (our current material-ui version does not offer withStyles )